src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const weekDiscount = 50;
+  const threeDaysDiscount = 20;
```

Comment:
the checklist, p.5

Reply:
Sorry, I dont know how I passed by checklist. Corrected everything